model/components/commercial/kloudfuse.yml

```diff
@@ -0,0 +1,29 @@
+---
+components:
+
+    # Kloudfuse
+  - id: kloudfuse
+    name: kloudfuse
+    description: Kloudfuse is a unified monitoring and observability platform covering metrics, traces and logs, events, etc. Kloudfuse supports all open source agents and data collection sources (prometheus, open telemetry, fluentbit, etc.) and also promql, logql and traceql.
+    link: https://www.kloudfuse.com/
+    github: []
```

Comment:
if there is no GitHub reference just remove the field altogether, just as within the definition for other commercial tools. E.g. AppDynamics also has no GitHub link in the component definition.